### PR TITLE
[native-pos]Update memory arbitrator config

### PR DESCRIPTION
**Pull request by xiaoxmeng**
Update memory config settings and enable the memory
arbitration by default and will turn on spilling by default
when spilling path setting is done.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java [hunk 1]
```diff
@@ -92,9 +92,9 @@ public class NativeExecutionSystemConfig
     private int systemMemoryGb = 10;
     private DataSize queryMemoryGb = new DataSize(systemMemoryGb, DataSize.Unit.GIGABYTE);
     private boolean useMmapAllocator = true;
-    private boolean enableMemoryArbitration = true;
-    private long memoryPoolInitCapacity = 512 << 20;
-    private long memoryPoolTransferCapacity = 256 << 20;
+    private String memoryArbitratorKind = "SHARED";
+    private long memoryPoolInitCapacity = 10 << 30;
```
Comment (tanjialiang) [inline presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java:96]: Do we want to leave a bit for pre-fetch?

**Review by tanjialiang — APPROVED** [review]

**Comment by tanjialiang** [comment]
Thanks for the change

File: presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java (hunk 1 above)
Comment (xiaoxmeng) [reply presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java:96]: Yeah, we could consider adding a config for the prefetch memory.

**Comment by shrinidhijoshi** [comment]
@xiaoxmeng Would this result in backward incompatibility between java and cpp code ? Currently cpp is released 4/5 times a day and java is release once a day, atleast for sapphire-velox.

**Comment by xiaoxmeng** [comment]
> @xiaoxmeng Would this result in backward incompatibility between java and cpp code ? Currently cpp is released 4/5 times a day and java is release once a day, atleast for sapphire-velox.

I will keep the old configs until we have removed those config use in meta internally. Thanks!

**Review by shrinidhijoshi — APPROVED** [review]
Thanks for the update. LGTM now.

**Pull request merged**